.prettierrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "printWidth": 80,
```

Comment:
Maybe 120? I've always find 80 so limitating

Reply:
Seams good to me, especially with the current wide screen monitors.
I will make a new commit.